graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/FederatedSchemaAutoConfiguration.kt

```diff
@@ -33,24 +34,29 @@ import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
 import java.util.Optional
 
 /**
- * SpringBoot autoconfiguration for generating Federated GraphQL schema.
+ * SpringBoot autoconfiguration for generating federated GraphQL schema object.
+ * These will override the beans in [NonFederatedSchemaAutoConfiguration] if federation is enabled.
  */
 @ConditionalOnProperty(value = ["graphql.federation.enabled"], havingValue = "true")
 @Configuration
+@Import(GraphQLExecutionConfiguration::class)
```

Comment:
I needed to create this new root `GraphQLExecutionConfiguration` so I could share the same beans in the renamed `NonFederatedSchemaAutoConfiguration`